src/algebra/group/opposite.lean

```diff
@@ -290,6 +290,18 @@ lemma units.coe_op_equiv_symm {M} [monoid M] (u : (Mˣ)ᵐᵒᵖ) :
   (units.op_equiv.symm u : Mᵐᵒᵖ) = op (u.unop : M) :=
 rfl
 
+lemma is_unit.op {M} [monoid M] {a : M} (h : is_unit a) : is_unit (op a) :=
+let ⟨a, _⟩ := h in ⟨units.op_equiv.symm (op a), by simpa⟩
+
+lemma is_unit.unop {M} [monoid M] {x : Mᵐᵒᵖ} (h : is_unit x) : is_unit (unop x) :=
+let ⟨a, _⟩ := h in ⟨unop a.op_equiv, by simpa⟩
+
+lemma is_unit_op_iff_is_unit {M} [monoid M] {a : M} : is_unit (op a) ↔ is_unit a :=
+⟨λ h, unop_op a ▸ h.unop, is_unit.op⟩
+
+lemma is_unit_unop_iff_is_unit {M} [monoid M] {x : Mᵐᵒᵖ} : is_unit (unop x) ↔ is_unit x :=
+⟨λ h, op_unop x ▸ h.op, is_unit.unop⟩
```

Comment:
These both look like good `@[simp]` lemmas to me

Reply:
Thanks! I added!